presto-main/src/main/java/com/facebook/presto/cost/EnsureStatsMatchOutput.java

```diff
@@ -29,16 +30,16 @@
     @Override
     public PlanNodeStatsEstimate normalize(PlanNode node, PlanNodeStatsEstimate estimate, Map<Symbol, Type> types)
     {
-        Map<Symbol, SymbolStatsEstimate> symbolSymbolStats = new HashMap<>();
-        estimate.getSymbolsWithKnownStatistics().stream()
-                .filter(node.getOutputSymbols()::contains)
-                .forEach(symbol -> symbolSymbolStats.put(symbol, estimate.getSymbolStatistics(symbol)));
+        PlanNodeStatsEstimate.Builder builder = buildFrom(estimate);
```

Comment:
It is required since we now remove extra stats instead of building output stats from ground up